release_notes/ocp_3_6_release_notes.adoc

```diff
@@ -0,0 +1,2419 @@
+----
+kind: StorageClass
+apiVersion: storage.k8s.io/v1beta1
+metadata:
+  name: fast
+provisioner: kubernetes.io/vsphere-volume
+parameters:
+    diskformat: zeroedthick
+    storagePolicyName: gold
+    fstype:     ext3
+----
+
+[[ocp-36-increased-security-with-iscsi-chap-mount-operations]]
+==== Increased Security with iSCSI CHAP and Mount Operations
+
+You can now use CHAP authentication for your iSCSI remote persistent volumes.
+Also, you can annotate your PVs to leverage any mount options that are supported
+by that underlying storage technology.
+
+The tenant supplies the correct user name and password for the CHAP
+authentication as a secret in their `podspec`. For mount options, you supply the
+annotation in the PV.
+
+----
+volumes:
+  - name: iscsivol
+    iscsi:
+      targetPortal: 127.0.0.1
+      iqn: iqn.2015-02.example.com:test
+      lun: 0
+      fsType: ext4
+      readOnly: true
+      chapAuthDiscovery: true
+      chapAuthSession: true
+      secretRef:
+         name: chap-secret
+----
+
+Set `volume.beta.kubernetes.io/mount-options` to
+`volume.beta.kubernetes.io/mount-options: rw,nfsvers=4,noexec`.
+
+See xref:../architecture/additional_concepts/storage.adoc#pv-mount-options[Mount
+Options] for more information.
+
+[[ocp-36-mount-options]]
+==== Mount Options (Technology Preview)
+
+You can now specify mount options while mounting a persistent volume by using
+the annotation `volume.beta.kubernetes.io/mount-options`
+
+See
+xref:../architecture/additional_concepts/storage.adoc#pv-mount-options[Persistent
+Storage] for more information.
+
+[[ocp-36-improved-automated-support-for-cns-backed-ocp-hosted-registry]]
+====  Improved and Fully Automated Support for CNS-backed OCP Hosted Registry
+
+Previously, only a few supported storage options existed for a scaled,
+highly-available integrated {product-title} (OCP) registry. Automated Container
+Native Storage (CNS) 3.6 and the {product-title} installer now include an option
+to automatically deploy a scale-out registry based on highly available storage,
+out of the box. When enabled in the installer’s inventory file, CNS will be
+deployed on a desired set of nodes (for instance, infrastructure nodes). Then,
+the required underlying storage constructs will automatically be created and
+configured for use with the deployed registry. Moving an existing registry
+deployment from NFS to CNS is also supported, and requires additional steps for
+data migration.
+
+The {product-title} registry backed with Container Native Storage (Gluster).
+This enables user to take advantage of the globally available storage capacity,
+strong read/write consistency, three-way replica, and RHGS data management
+features.
+
+The feature is provided through integrations in the {product-title}
+xref:../install_config/install/advanced_install.adoc#install-config-install-advanced-install[advanced
+installation] process. A simple change to the inventory file is all that is
+required.
```

Comment:
Above should specify that dedicated storage devices are required. e.g. `A few dedicated storage devices and a simple change to the inventory file`.